docs/touchablewithoutfeedback.md

```diff
@@ -219,6 +219,46 @@ See the [Accessibility guide](accessibility.md#accessibilityvalue-ios-android) f
 
 ---
 
+### `aria-valuemax`
+
+It Represents the maximum value for range-based components, such as sliders and progress bars. Has precedence over the `accessibilityValue.max` prop.
```

Comment:
```suggestion
Represents the maximum value for range-based components, such as sliders and progress bars. Has precedence over the `max` value in the `accessibilityValue` prop.
```

Please remove "It" at the beginning of each description, ensure that case is correct and rephrase the precedence sentence as seen above in other entries added.